### DISCUSSION TITLE
Cables of Cables in Rings -- a Best Practices query

### DISCUSSION THREAD

**Discussion by postilion** [netbox-community/netbox#6643] [Q&A]
Thanks for Netbox, love it!
We've got a few sites and use fiber rings to connect IDFs to MDF in each, with one or more ring per site.  For example, at our main site we have one ring which connects MDF to two IDFs which house distribution switches in addition to access switches.  Then a separate ring connects those two IDFs to the rest of the IDFs, which hold just access switches.

Per [earlier](https://github.com/netbox-community/netbox/discussions/5749) [discussions](https://groups.google.com/g/netbox-discuss/c/8EtW5u4fpOM/m/8qWoWWBKBQAJ) I understand the one-rear-port-to-many-front-port method for associating strands to cables; that much is clear.  What I'm curious about is how people handle the strands which bypass a mid-sheath cut and just continue to the next IDF?  

I could just model a ring as several separate cables from MDF to IDFs, or from one IDF to many IDFs, but that elides the fact that these are all actually bundled into one physical cable.

Again, while I can see ways to do this, I am curious to hear how others have tackled this seemingly (to me at least) common situation?

Thanks in advance,
-nic

**Comment by julianze** [comment]
> What I'm curious about is how people handle the strands which bypass a mid-sheath cut and just continue to the next IDF?

> `I could just model a ring as several separate cables from MDF to IDFs, or from one IDF to many IDFs, but that elides the fact that these are all actually bundled into one physical cable.

Do you want to patch through passivley the part of a Strands (single port of trunk cable) to another IDF?
